config/options.go

```diff
@@ -328,6 +330,13 @@ var defaultOptions = Options{
 	EnvoyAdminAccessLogPath:             os.DevNull,
 	EnvoyAdminProfilePath:               os.DevNull,
 	ProgrammaticRedirectDomainWhitelist: []string{"localhost"},
+
+	RuntimeFlags: DefaultRuntimeFlags(),
```

Comment:
I think you were right that there is some subtlety around default flags. When I tried this out locally, I did not observe the default flag value in the RuntimeFlags map after config parsing.

I think maybe the initialization of RuntimeFlags should be moved into the NewDefaultOptions() function, so we get a distinct map instance each time NewDefultOptions() is called?